testsuite/features/secondary/srv_check_dist_channel_mapping_page.feature

```diff
@@ -0,0 +1,133 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Check if Distribution Channel Mapping works correctly
+
+    Scenario: Log in as admin user
+        Given I am authorized for the "Admin" section
+
+    Scenario: Check if Distribution Channel Mapping page exists
+        When I follow the left menu "Software > Distribution Channel Mapping"
+        Then I should see a "Distribution Channel Mapping" text
+        And I should see a "Channel List" link in the left menu
+        And I should see a "Package Search" link in the left menu
+        And I should see a "Manage" link in the left menu
+        And I should see a "Distribution Channel Mapping" link in the left menu
+        And I should see a "Create Distribution Channel Mapping" link
+        And I should see a "No distribution channel mappings currently exist." text in the content area
+
+    Scenario: Create new map for x86_64 suse clients
+        When I am on the "Distribution Channel Mapping" page
```

Comment:
I think it probably is better to repeat `When I follow the left menu "Software > Distribution Channel Mapping"` every time 
If the previous scenario fails and the test did not redirect to this page, the scenario will automatically fail.

Reply:
Which would make the definition of this associated test not necessary

Reply:
You mean have `When I follow the left menu "Software > Distribution Channel Mapping"` in every scenario?